spring-pulsar-docs/src/main/asciidoc/pulsar.adoc

```diff
@@ -1147,7 +1146,9 @@ void listen1(String message) {
 ----
 ====
 
-When running this app, you will see the following output:
+TIP: The id parameter is the value of the @PulsarListener id attribute (when pausing a @PulsarListener).
```

Comment:
**[TinyNit]** Don't hate me, but do you mind formatting `@PulsarListener` w/ "` `"?